### DISCUSSION TITLE
Leaflet Draw Integration

### DISCUSSION THREAD

**Discussion by t1a2l** [Leaflet/Leaflet#8922] [Ideas]
I suggest adding leaflet draw into leaflet as a core functionality, it seems the draw package is not maintained and no one answer to people issues, or check their commits for the draw package. 
By adding it to the core it will continue to be maintained and add more new features.

**Comment by Falke-Design** [comment]
Thank you for your suggestion but we already on our limit to maintain Leaflet. We can't take care of another library too. Also we want to keep Leaflet lightweight and everything else should be a plugin, like Leaflet-Draw.

For more informations take a look into #7704

Maybe you want to try out a differnet library then Leaflet-Draw: https://leafletjs.com/plugins.html#edit-geometries
